src/main/java/org/apache/sling/feature/cpconverter/cli/ContentPackage2FeatureModelConverterLauncher.java

```diff
@@ -145,6 +146,9 @@ public final class ContentPackage2FeatureModelConverterLauncher implements Runna
 
     @Option(names = { "--sling-initial-content-policy" }, description = "Determines what to do with Sling-Initial-Content found in embedded bundles. Valid values: ${COMPLETION-CANDIDATES}.", required = false, showDefaultValue = Visibility.ALWAYS)
     private SlingInitialContentPolicy slingInitialContentPolicy = SlingInitialContentPolicy.KEEP;
+    
+    @Option(names = { "--runmode-policy" }, description = "Determines how to determine the final runmode of an artifac. Valid values: ${COMPLETION-CANDIDATES}.", required = false, showDefaultValue = Visibility.ALWAYS)
```

Comment:
The readme needs to be adjusted. To me this option requires further description, as I don't get its meaning.

Reply:
updated

Reply:
rather `--run-mode-policy`. Also I still don't see the readme in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/blob/master/README.md#run-modes and https://github.com/apache/sling-org-apache-sling-feature-cpconverter/blob/master/README.md#the-cli-tool updated.